src/listeners/substrate-listener/commands/orders/order-fulfilled/order-fulfilled.handler.ts

```diff
@@ -152,12 +152,27 @@ export class OrderFulfilledHandler
         };
         await this.rewardService.insert(dataCustomerLoggingInput);
 
+        // Write Logging Notification Lab Reward From Request Service
+        const labNotificationInput: NotificationDto = {
+          role: 'Lab',
+          entity_type: 'Reward',
+          entity: 'OrderFulfilled',
```

Comment:
entity should be `'Request Service Staking '`
![image](https://user-images.githubusercontent.com/37687466/172154005-cfabcd77-7a4c-438c-aa7e-63330bba52eb.png)